.pre-commit-config.yaml

```diff
@@ -1,7 +1,7 @@
 exclude: "(.*\\.svg)|(.*\\.qmd)|(.*\\.ambr)|(.*\\.csv)|(.*\\.txt)|(.*\\.json)"
 repos:
   - repo: https://github.com/pycqa/flake8
-    rev: 6.0.0
+    rev: 7.1.1
```

Comment:
I had to update flake8 because the used version seemed to have a bug complaining about the content of f-strings:

```
flake8...................................................................Failed
- hook id: flake8
- exit code: 1

great_tables/_tbl_data.py:803:70: E231 missing whitespace after ':'
great_tables/_tbl_data.py:809:33: E221 multiple spaces before operator
```